zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -92,6 +92,15 @@ object Response {
     }
   }
 
+  /**
+   * Creates a response with content-type set to text/html
+   */
+  def html(data: String): UResponse =
+    Response(
+      data = HttpData.fromChunk(Chunk.fromArray(data.getBytes(HTTP_CHARSET))),
```

Comment:
```suggestion
      data = HttpData.fromText(Chunk.fromArray(data.getBytes(HTTP_CHARSET))),
```

Can keep `charset` as an argument to `html()` defaulting to `HTTP_CHARSET`.